examples/xgboost/utils/prepare_job_config.py

```diff
@@ -26,8 +26,9 @@
 
 def job_config_args_parser():
     parser = argparse.ArgumentParser(description="generate train configs for HIGGS dataset")
-    parser.add_argument("--data_split_path", type=str, default="./data_splits", help="Path to data split folder")
+    parser.add_argument("--data_root", type=str, default="/tmp/nvflare/xgboost_higgs_dataset", help="Path to dataset")
```

Comment:
Path to dataset config json files for each client? (dataset path is "~/dataset/HIGGS.csv")

Reply:
thanks